eth/config.go

```diff
@@ -94,6 +94,7 @@ type Config struct {
 	SkipBcVersionCheck bool `toml:"-"`
 	DatabaseHandles    int  `toml:"-"`
 	DatabaseCache      int
+	DatabaseFreezer    string
```

Comment:
minor: `-er` suffix in Go usually implies an interface.  When I first read through this code I was expecting Freezer to be an interface w/ a `Freeze` method, didn't realize until later that it was referring to the "colder portion of a refrigerator" :)

Reply:
Agree it should probably end in `Fridge` :)